repo-scripts/size-analysis/cli.ts

```diff
@@ -0,0 +1,93 @@
+/**
+ * @license
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as yargs from 'yargs';
+import { run as runBundleAnalysis } from './bundle-analysis';
+import { analyzePackageSize } from './package-analysis';
+
+// eslint-disable-next-line no-unused-expressions
+yargs
+  .command(
+    '$0',
+    'Analyze the size of individual exports from packages',
+    {
+      inputModule: {
+        type: 'array',
+        alias: 'im',
+        desc:
+          'The name of the module(s) to be analyzed. example: --inputModule "@firebase/functions-exp" "firebase/auth-exp"'
+      },
+      inputDtsFile: {
+        type: 'string',
+        alias: 'if',
+        desc: 'support for adhoc analysis. requires a path to dts file'
```

Comment:
Is this a `d.ts` file or is `dts` something else?

Reply:
`d.ts` file. Updated to `d.ts` to make it clear.